include/auth.php

```diff
@@ -19,8 +19,10 @@
 			}
 
 			// Renew the cookie
-			// Expires after 90 days - TODO: use a configuration variable
-			new_cookie(90*24*60*60, $r[0]);
+			// Expires after 7 days by default,
+			// can be set via system.auth_cookie_lifetime
+			$authcookiedays = get_config('system','auth_cookie_lifetime') || 7;
```

Comment:
`function get_config` is deprecated. Please use `Config::get()` instead.
This would mean adding `use \Friendica\Core\Config;` at the top of file before `require_once`.
This line would change into `$authcookiedays = Config::get('system', 'auth_cookie_lifetime') || 7;`

STANDARDS: please add a space after comma in function call